File: lab1/Row.php
<?php

class Row
{
    private $x;
    private $y;
    private $R;
    private $isHit;
    private $currentTime;
    private $workTime;

    public function __construct($currentTime, $x, $y, $R, $isHit, $workTime)
    {
        $this->currentTime = $currentTime;

        $this->x = $x;
        $this->y = $y;
        $this->R = $R;

        $this->isHit = $isHit;
        $this->workTime = $workTime;
    }

    public function getData()
    {
        return "<tr> <td>$this->currentTime</td> <td>$this->x</td> <td>$this->y</td>
          <td>$this->R</td> <td>$this->isHit</td> <td>$this->workTime с</td>
            </tr>";
    }
}

File: lab2/src/main/java/utilities/Row.java
package utilities;

public class Row {
    private final float x;
    private final float y;
    private final float R;
    private final String isHit;
    private final String currentTime;
    private final long workTime;

    public Row(String currentTime, float x, float y, float R, String isHit, long workTime) {
        this.currentTime = currentTime;
        this.x = x;
        this.y = y;
        this.R = R;
        this.isHit = isHit;
        this.workTime = workTime;
    }

    public String getData() {
        return "<tr> <td>" + currentTime + "</td> <td>" + x + "</td> <td>" + y + "</td> " +
                "<td>" + R + "</td> <td>" + isHit + "</td> <td>" + workTime + "ms</td> " +
                "</tr>";
    }
}
